README.md

```diff
@@ -171,43 +171,42 @@ The *Insights Dashboard* provides an overview of the models that OpenScale is mo
 
 ![Deploy OpenScale](doc/source/images/aios-deploy-service.png)
 
-* When the dashboard loads, Click on the 'Model Monitors'  tab and you will see the one deployment you configured in the previous section.
+* When the dashboard loads, Click on the `Model Monitors`  tab and you will see the one deployment you configured in the previous section.
 
 ![OpenScale Insight Dashboard Tile Open](doc/source/images/OpenScaleInsightDashTileOpen.png)
 
 Do not worry if the name you see does not match exactly with the screenshot. The deployment name you see will correspond to the variable used in the Jupyter notebook
 
-You will see the triangle with `!` under `Fairness` -> `Sex`. This indicates that there has been an alert for the `Fairness` monitor. Alerts are configurable, based on thresholds for fairness outcomes which can be set and altered as desired.
+![OpenScale Quality Monitor](doc/source/images/OpenScaleQualityMonitor.png)
 
-* By moving your mouse pointer over the graph, you can see the values change, and which contains bias. Click one spot to view the details. Later, we'll click `Configure Monitors` to get a Fairness endpoint:
+The quality monitor scans the requests sent to your model deployment \(i.e the payload\) to let you know how well your model predicts outcomes. Quality metrics are calculated hourly, when OpenScale sends manually labeled feedback data set to the deployed model.
```

Comment:
"i.e" is missing a period after the e.  It is usually followed by a comma as well, but not sure that is better.
Not sure about the last sentence it just sounds a little off to me.  Changing "feedback data set" to just "feedback data" or just "feedback" would help.  Alternatively, if you want to keep "feedback data set" then I'd either make it plural "data sets" or keep it singular and just add "a" or before "manually labelled". Not sure if the grammar is wrong or if that is just my preference/style.

Reply:
fixed.